### PR TITLE
BETA: Register codecloak.jigar.is-a.dev

### PR DESCRIPTION
**Pull request by drone911**
Added `codecloak.jigar.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by drone911** [comment]
I am sorry, I did the temporary.is-a.dev as a test. I just want codecloak.jigar.is-a.dev.

File: domains/codecloak.jigar.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "drone911",
+        "email": "jigar1822@gmail.com"
+    },
+    "record": {
+        "URL": "codecloak.onrender.com"
```
Comment (Omxpro) [inline domains/codecloak.jigar.json:7]: URL records must have https:// and your other domain should be deleted

**Review by Omxpro — CHANGES_REQUESTED** [review]

**Review by Omxpro — APPROVED** [review]

**Review by iostpa — CHANGES_REQUESTED** [review]
jigar.is-a.dev doesnt exist, so you cant make a nested domain with it

**Comment by drone911** [comment]
I added the domain jigar.is-a.dev. I hope that is okay.

**Review by Omxpro — APPROVED** [review]

**Review by iostpa — APPROVED** [review]

**Pull request merged**